lib/foodcritic/command_line.rb

```diff
@@ -76,7 +77,8 @@ def initialize(args)
           @options[:version] = true
         end
         opts.on('-X', '--exclude PATH',
-                'Exclude path(s) from being linted.') do |e|
+                'Exclude path(s) from being linted. Default test/**/*,'\
+                'spec/**/*,features/**/*') do |e|
```

Comment:
Even that would just add `none` to the existing list I think. At a minimum we should probably make it so passing any option skips the defaults, that would at least make the workaround you mention an option.